server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -92,14 +110,16 @@ public TabletsMutator mutateTablets() {
 
   @Override
   public ConditionalTabletsMutator conditionallyMutateTablets() {
-    return new ConditionalTabletsMutatorImpl(context, getTableMapper());
+    return new ConditionalTabletsMutatorImpl(context, getTableMapper(), sharedMetadataWriter,
+        sharedUserWriter);
```

Comment:
If not storing a ref to the writers, then seems like could do the following.

```suggestion
    return new ConditionalTabletsMutatorImpl(context, getTableMapper(), context.getSharedMetadataWriter(), context.getSharedUserWriter());
```